shared-bindings/supervisor/Runtime.c

```diff
@@ -45,10 +48,32 @@
 //|         ...
 //|
 
+//|     usb_connected: bool
+//|     """Returns the CIRCUITPY drive mount status (read-only)."""
```

Comment:
I don't think we want this tied to the drive status at all. Instead it should be true whenever USB is communicating at all. I think this is `tud_mounted` (which has nothing to do with MSC). Please don't delay in this method either. Calling code can do it if it wants.

Reply:
Should I rename this function to `drive_mounted` ?

Reply:
No, I don't think we should expose that until we need it. Instead, let's change how this works.

Reply:
Currently I use this for checking usb in deep sleep PR.

Reply:
You shouldn't use the MSC connection to know when to sleep. Instead, it should be if enumerated at all. TinyUSB calls this being "mounted" I think.